SECURITY.md

```diff
@@ -0,0 +1,41 @@
+<!-- BEGIN MICROSOFT SECURITY.MD V0.0.8 BLOCK -->
```

Comment:
Would recommend not adding a new security file, especially as it's not in the other vscode-remote-try-* repos. Unless it's something we need to add in all of those as well.

Reply:
I think it's now mandatory. We even have bots pushing to add this file to all OSS repos (even on this repo: https://github.com/microsoft/vscode-remote-try-dotnetcore/pull/25)

I would recommend to add it to all our repos

Reply:
Got it, thanks for sharing. It looks like it wasn't automatically PR'd into our other remote-try repos (for instance https://github.com/microsoft/vscode-remote-try-python/pulls) - would your team be able to help with these too?

Reply:
Done :)